docs/components/customization/localization.mdx

```diff
@@ -73,9 +74,22 @@ function MyApp({ Component, pageProps }: AppProps) {
 export default MyApp;
 ```
 
+## Adding or updating a localization 
+
+Our localizations are customer sourced and we encourage customers to add or update localizations. The process is:
+
+1. Fork the [https://github.com/clerkinc/javascript/](https://github.com/clerkinc/javascript/) repo.
+2. Clone it locally to edit it.
+3. Review our [Contributing](https://github.com/clerkinc/javascript/blob/main/docs/CONTRIBUTING.md) guide.
+4. If you are updating an existing localization locate the file in `packages/localizations/src`
+5. If you are adding a new language copy the en-US.ts file and name it according to your language. The naming is the abbreciated language-region. For French in Canada it would be fr-CA.ts.
+6. Go through the file and edit the entries.
+7. If you are adding a new localization, add the language to to the `packages/localizations/src/index.ts` file.
+8. Commit your changes to git, push them to your fork. Create a Pull Request from your fork to Clerk's repo againat the `main` branch. We will review and either approve or ask for updates.
+
 ## Custom localizations
 
-You can also provide your own localizations for the Clerk Components. This is useful if you want to provide localization for a language that Clerk doesn't currently support or if you want to change the wording to suit your brand.
+You can also provide your own localizations for the Clerk Components. This is useful if you want to provide limited or quick localization for a language that Clerk doesn't currently support or if you want to change the wording to suit your brand.
```

Comment:
```suggestion
You can also provide your own localizations for the Clerk Components. This is useful if you want to provide limited or quick localization for a language that Clerk doesn't currently support, or if you want to change the wording to suit your brand.
```

Reply:
Opening linear to discuss further, will merge this version for the time being.